apps/api/v2/src/modules/organizations/inputs/create-organization-user.input.ts

```diff
@@ -0,0 +1,17 @@
+import { CreateUserInput } from "@/modules/users/inputs/create-user.input";
+import { MembershipRole } from "@prisma/client";
+import { IsString, IsOptional, IsBoolean, IsEnum } from "class-validator";
+
+export class CreateOrganizationUserInput extends CreateUserInput {
+  @IsOptional()
+  @IsString()
+  locale = "en";
+
+  @IsOptional()
+  @IsEnum(MembershipRole)
+  organizationRole: MembershipRole = MembershipRole.MEMBER;
+
+  @IsOptional()
+  @IsBoolean()
+  autoAccept = true;
```

Comment:
Assume that if we are creating org users via API we want the user to be automatically accepted into the org.